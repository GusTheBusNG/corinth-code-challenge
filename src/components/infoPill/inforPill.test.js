import React from 'react';
import { create } from 'react-test-renderer';

import InfoPill from './';

const createInfoPill = () => 
  create(<InfoPill icon='lightsaber'>and the children too</InfoPill>);


describe('InfoPill', () => {
  describe('render', () => {
    it('renders correctly', () => {
      expect(createInfoPill().toJSON()).toMatchSnapshot();
    })
  })
})
