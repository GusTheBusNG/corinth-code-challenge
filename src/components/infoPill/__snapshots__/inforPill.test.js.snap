// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoPill render renders correctly 1`] = `
<div
  className="info-pill"
>
  <img
    alt="lightsaber"
    className="icon"
    src="https://res.cloudinary.com/dnglix0tu/image/upload/f_auto/v1/LifeWay-star-wars/lightsaber?_a=ATAMhAA0"
  />
  <p
    className="content"
  >
    and the children too
  </p>
</div>
`;
